builds/e2e/templates/e2e-setup.yaml

```diff
@@ -169,6 +169,7 @@ steps:
         isa95Tag = 'false';
         edgeAgentImage = "$edgeAgentImage";
         edgeHubImage = "$edgeHubImage";
+        edgeHubSchemaVersion = "1.2";
```

Comment:
Should SchemaVersion be another permutation variable? Or that's out of scope for the moment?

Reply:
#2 seems reasonable to me

Reply:
Ok, I made the change to have all the scenarios use the 1.1 EH schema and all the tests still pass.